nautobot/utilities/tests/test_filters.py

```diff
@@ -991,6 +992,48 @@ def test_device_comments_multiple_value_charfield_iregex_negation(self):
         self.assertEqual(self.DeviceFilterSetWithComments(params, self.device_queryset).qs.count(), 0)
 
 
+class GetFiltersetTestValuesTest(FilterTestCases.BaseFilterTestCase):
+    """Tests for `BaseFilterTestCase.get_filterset_test_values()`."""
+
+    queryset = Site.objects.all()
+    exception_message = "Cannot find valid test data for Site field description"
+
+    @classmethod
+    def setUpTestData(cls):
+        statuses = Status.objects.get_for_model(Site)
+        cls.status_active = statuses.get(slug="active")
+
+        Site.objects.all().delete()
+
+    def test_empty_queryset(self):
+        with self.assertRaisesMessage(ValueError, self.exception_message):
+            self.get_filterset_test_values("description")
+
+    def test_object_return_count(self):
+        for n in range(1, 11):
+            Site.objects.create(name=f"Site{n}", status=self.status_active, description=f"description {n}")
+        test_values = self.get_filterset_test_values("description", Site.objects.all())
+        self.assertNotEqual(test_values, 0)
+        self.assertNotEqual(test_values, Site.objects.count())
```

Comment:
`test_values` is a list, right? These assertions seem incorrect to me if we're comparing a list against an int.

Reply:
Sure is! Thanks for catching that